<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Natice_alarm</class>
 <widget class="QWidget" name="Natice_alarm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>454</width>
         <height>254</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>454</width>
         <height>254</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">width:454px;
height:218px;
background:rgba(14,19,22,1);
opacity:0.95;
border-radius: 6px;
</string>
       </property>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>424</x>
          <y>0</y>
          <width>30</width>
          <height>30</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-image: url(:/image/1x/close_light.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>177</x>
          <y>10</y>
          <width>100</width>
          <height>23</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>闹钟</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>64</y>
          <width>454</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 25 14pt  &quot;Noto Sans Devanagari Md&quot;;</string>
        </property>
        <property name="text">
         <string>工作会议</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>94</y>
          <width>454</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: 28pt &quot;Noto Sans Coptic&quot;;</string>
        </property>
        <property name="text">
         <string>10 : 45</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>97</x>
          <y>169</y>
          <width>100</width>
          <height>32</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
font: 11pt ;
color: rgb(255, 255, 255);
background:rgba(46,46,46,1);
border-radius:4px;}
QPushButton:hover{background:rgba(143, 97, 47, 0.1)}</string>
        </property>
        <property name="text">
         <string>稍后提醒</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>257</x>
          <y>169</y>
          <width>100</width>
          <height>32</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{color: rgb(255, 255, 255);
background-color: rgba(39,207,129,0.9);
 border-radius:4px;
font: 11pt ;}
QPushButton:hover{background-color: rgb(27, 143, 89);}
</string>
        </property>
        <property name="text">
         <string>关闭</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>221</y>
          <width>454</width>
          <height>23</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>454</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
